packages/e2e-tests/specs/editor/various/list-view.test.js

```diff
@@ -82,11 +82,12 @@ describe( 'List view', () => {
 		await page.keyboard.press( 'Enter' );
 
 		const canvasImageBlock = await page.waitForSelector(
-			'figure[aria-label="Block: Image"]'
+			'button.block-editor-media-placeholder__upload-button'
 		);
 		expect( canvasImageBlock ).toHaveFocus();
```

Comment:
It'd be good to rename the variable to `uploadButton`. There's also a preference on the project for using labels in selectors, though here that would involve an xpath selector. Something like this:
```js
const uploadButton = await page.waitForXPath(
	'//button[.="Upload"]'
);
```

Reply:
I had no idea how to do that so I matched against the class. I thought I had to grab innerText or something. I've never seen xpath syntax before.

Reply:
I further refined your xpath syntax as the one you suggested failed. Does it look good?

Reply:
Looks good 👍 

Xpaths have a tricky syntax, but sometimes they're the best option.